ScreenManager.c

```
@@ -260,12 +272,20 @@ void ScreenManager_run(ScreenManager* this, Panel** lastFocus, int* lastKey, con
    this->name = name;
 
    while (!quit) {
+      int statusBarY2 = ScreenManager_shouldShowStatusBar(this) ? -2 : -1;
+      if (this->y2 != statusBarY2) {
+         this->y2 = statusBarY2;
+         ScreenManager_resize(this);
```

Cppcheck (2.21.0)

[style] The function 'Process_getGroupOrParent' is never used.

(unusedFunction)